recipe_scrapers/weightwatchers.py

```diff
@@ -9,7 +9,7 @@
 class WeightWatchers(AbstractScraper):
     @classmethod
     def host(cls):
-        return "www.weightwatchers.com"
+        return "weightwatchers.com"
```

Comment:
We should update the test data for `weightwatchers` if we're doing this; yep, it's probably a bug that it has been using a default `schema.org` scraping method -- but it's probably safer to retain compatibility at this point, because the `weightwatchers` scrapers (incl. public) haven't been significantly updated since Y2022.

Reply:
@jayaddison To clarify, what do I need to update?